packages/rocketchat-ui-sidenav/side-nav/listDirectMessagesFlex.js

```diff
@@ -39,6 +39,12 @@ Template.listDirectMessagesFlex.events({
 		SideNav.leaveArrow();
 	},
 
+	'keydown #channel-search': function(e, t) {
+		if (e.keyCode == 13) {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Expected '===' and instead saw '=='.](https://www.codacy.com/app/RocketChat/Rocket-Chat/file/2513052305/issues/source?bid=3199634&fileBranchId=3277287#l43)